CHANGELOG.md

```diff
@@ -193,55 +217,60 @@ Thus it was decided that it was not worth it to keep backward compatibility and
 ```
 
 ## [1.1.3] - 25 / 05 / 2021
-* fix initial value not setting up correctly
-* Added different types of Country selector display: SelectorDisplay.coversBody, SelectorDisplay.coversLower. The old value is coversLower and the new default value is coversBody. 
+
+- fix initial value not setting up correctly
+- Added different types of Country selector display: SelectorDisplay.coversBody, SelectorDisplay.coversLower. The old value is coversLower and the new default value is coversBody.
 
 ## [1.1.2] - 20 / 05 / 2021
-* minor changelog fix.
+
+- minor changelog fix.
 
 ## [1.1.1] - 20 / 05 / 2021
-* Added key to exported widgets.
+
+- Added key to exported widgets.
 
 ## [1.1.0] - 19 / 05 / 2021
-* [Breaking] : some classes from the package phone_numbers_parser aren't exported anymore. Mostly classes that are not intended to be used with this package.
-* Added example for widgets FlagDialCodeChip and CountrySelector
-* Added default values for FlagDialCodeChip and removed some unused values.
+
+- [Breaking] : some classes from the package phone_numbers_parser aren't exported anymore. Mostly classes that are not intended to be used with this package.
+- Added example for widgets FlagDialCodeChip and CountrySelector
+- Added default values for FlagDialCodeChip and removed some unused values.
 
 ## [1.0.2] - 23 / 04 / 2021
-* Upped dependency phone_numbers_parser to 0.1.3
-* Added phoneNumberType input to validate against specific types (mobile, fixed line)
+
+- Upped dependency phone_numbers_parser to 0.1.3
+- Added phoneNumberType input to validate against specific types (mobile, fixed line)
 
 ## [1.0.1] - 23 / 04 / 2021
-* Fixed validity issue (issue #1)
-* Allow country to be searched by dial code (issue #2)
-* Sort country search results to have more meaningful results at the top
-* Upped dependency phone_numbers_parser to 0.1.0
+
+- Fixed validity issue (issue #1)
```

Comment:
What does those change do ?

Reply:
I rhink spaces only removed
And add what change in new version

Reply:
the * was changed to -

Reply:
i am not change it i take code from dev channel and add my code

Reply:
This page indicates that this was changed on your side, maybe by some formatter.